cmd/genji/shell/shell.go

```diff
@@ -189,12 +195,34 @@ func (sh *Shell) runExecutor(ctx context.Context, promptExecCh chan string) erro
 			}
 			if err != nil {
 				fmt.Println(err)
+				continue
+			}
+
+			// if showtime is true, ensure it's a query, and it was executed.
+			if showTime && !strings.HasPrefix(input, ".") && strings.HasSuffix(input, ";") {
+				displayTime(start)
 			}
 		case promptExecCh <- "":
 		}
 	}
 }
 
+// displayTime shows execution time of a query.
+func displayTime(start time.Time) {
```

Comment:
How about using the default display of the [time.Duration](https://pkg.go.dev/time#Duration.String) type?

Reply:
I think we can just use the time.Duration output yeat

Reply:
Simple and efficient!